### PR TITLE
Add Vietnamese translation

### PR DESCRIPTION
**Pull request by tingyun63**
Vietnamese Language

### CODE REVIEW

File: files/lang/vi.po [hunk 1]
```diff
@@ -0,0 +1,9623 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: fheroes2\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-07 14:32+0000\n"
+"PO-Revision-Date: 2023-03-10 00:04+0700\n"
+"Last-Translator: fheroes2 team <fhomm2@gmail.com>\n"
+"Language-Team: \n"
+"Language: uk\n"
```
Comment (Mr-Bajs) [inline files/lang/vi.po:13]: Languange uk, ukrainian?

File: files/lang/vi.po [hunk 2]
```diff
@@ -0,0 +1,9623 @@
+# Ukrainian translation of fheroes2
```
Comment (Mr-Bajs) [inline files/lang/vi.po:1]: Ukrainian?

File: files/lang/vi.po (hunk 1 above)
Comment (Nad-SK) [reply files/lang/vi.po:13]: It's the first PR for the vi.po only. Just copy from uk.po

File: files/lang/vi.po (hunk 2 above)
Comment (Nad-SK) [reply files/lang/vi.po:1]: It will be fix in the next PR.

Comment (Nad-SK) [reply files/lang/vi.po:1]: Approve for the first 30 lines (maybe more) translation. @zenseii

Comment (ihhub) [reply files/lang/vi.po:1]: This must be fixed.

File: files/lang/vi.po (hunk 1 above)
Comment (ihhub) [reply files/lang/vi.po:13]: This must be fixed.

**Comment by ihhub** [comment]
Hi @tingyun63 , please update the title of this pull request as well as address given comments. Also please remove any leftovers from Ukrainian translation. Copy-paste mistakes are the worst to handle in the future.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@tingyun63 , thank you very much for your contribution to the project!

**Pull request merged**